packages/ng-primitives/select/src/select-portal/select-portal-state.ts

```
@@ -0,0 +1,87 @@
+  show(): Promise<void>;
+
+  /**
+   * Hide the portal.
+   * @internal
+   */
+  hide(): Promise<void>;
+}
+
+export interface NgpSelectPortalProps {
+  // No props needed, injected from context
+}
```

Biome (2.4.4)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)